static/js/protein/chart.ts

```diff
@@ -42,12 +91,72 @@ export function drawTissueScoreChart(
   svg
     .append("g")
     .selectAll("rect")
-    .data(tmpData)
+    .data(reformattedData.sort((a, b) => d3.descending(a.value, b.value)))
     .enter()
     .append("rect")
-    .attr("x", (d) => d.x1)
-    .attr("y", (d) => d.y1)
-    .attr("width", (d) => d.x2 - d.x1)
-    .attr("height", (d) => d.y2 - d.y1)
-    .attr("fill", "teal");
+    .attr("x", x(0))
+    .attr("y", (d) => y(d.name))
+    .attr("width", (d) => x(d.value) - x(0))
+    .attr("height", y.bandwidth())
+    .style("fill", function (d) {
+      if (d.name == "Eye") {
+        return "green";
+      } else if (d.name == "Cartilage") {
+        return "yellow";
+      } else if (d.name == "Liver") {
+        return "steelblue";
+      } else {
+        return "red";
+      }
+    })
+    .on("mouseover", function (d, i) {
```

Comment:
nit: neither d or i are used so replace with just ()
nit: function should have a type definition, so here it would be

so this should look like & same for all the other mouseover and mouseout functions (line 115, 145, 148, 156, 159):
`function (): void {...`